IndexError in case of empty texts

I have this error occurring in my code in case I call `adjust_text` with an empty list of text objects. 

```
        plt.draw()
        ax = ax or plt.gca()
        r = get_renderer(ax.get_figure())
>       transform = texts[0].get_transform()
E       IndexError: list index out of range
/usr/local/lib/python3.7/site-packages/adjustText/__init__.py:567: IndexError
```

The error didn't occur in v0.7.3 but it does now after I updated adjustText to v0.8. 

It's an easy fix on my side, but you may want to handle the case of client code supplying an empty list of texts.

--- a/adjustText/__init__.py
+++ b/adjustText/__init__.py
@@ -71,7 +71,7 @@ def get_orig_coords(transform, t_x, t_y):
 
 def get_bboxes(objs, r=None, expand=(1, 1), ax=None, transform=None):
     """
-    
+
 
     Parameters
     ----------
@@ -561,8 +561,10 @@ def adjust_text(
     int
         Number of iteration
     """
-    plt.draw()
+    if not texts:
+        return 0
     ax = ax or plt.gca()
+    ax.figure.draw_without_rendering()
     r = get_renderer(ax.get_figure())
     transform = texts[0].get_transform()
     if (x is not None) & (y is not None):

